### PR TITLE
Orchestrator database

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-490

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -30,3 +30,104 @@ resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
...
+
+module "bulk-scan-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product            = var.product
+  component          = var.component
+  location           = var.location_db
+  env                = var.env
+  database_name      = "bulk_scan"
+  postgresql_user    = "bulk_scanner"
+  postgresql_version = "10"
```
Comment (mdayican) [inline infrastructure/main.tf:42]: should be latest,  I guess 11

File: infrastructure/main.tf [hunk 2]
```diff
@@ -30,3 +30,104 @@ resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
...
   key_vault_id = data.azurerm_key_vault.key_vault.id
 }
+
+module "bulk-scan-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product            = var.product
+  component          = var.component
+  location           = var.location_db
+  env                = var.env
+  database_name      = "bulk_scan"
```
Comment (mdayican) [inline infrastructure/main.tf:40]: sth with orchestrator?

File: infrastructure/main.tf [hunk 3]
```diff
@@ -30,3 +30,104 @@ resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
...
+  postgresql_user    = "bulk_scanner"
```
Comment (mdayican) [inline infrastructure/main.tf:41]: sth with orchestrator?

File: infrastructure/main.tf [hunk 4]
```diff
@@ -30,3 +30,104 @@ resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
...
+
+# Staging DB to be used by AAT staging pod for functional tests
+module "bulk-scan-staging-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  name               = "${var.product}-${var.component}-staging"
+  product            = var.product
+  component          = var.component
+  location           = var.location_db
+  env                = var.env
+  database_name      = "bulk_scan"
```
Comment (mdayican) [inline infrastructure/main.tf:57]: same here, sth name and user with orchestrator?

File: infrastructure/main.tf [hunk 5]
```diff
@@ -30,3 +30,104 @@ resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
...
 }
+
+module "bulk-scan-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product            = var.product
+  component          = var.component
+  location           = var.location_db
+  env                = var.env
+  database_name      = "bulk_scan_orchestrator"
+  postgresql_user    = "bulk_scanner_orch"
```
Comment (mdayican) [inline infrastructure/main.tf:41]: we can use same "bulk_scan_orchestrator" ?

File: infrastructure/main.tf [hunk 6]
```diff
@@ -30,3 +30,104 @@ resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
...
+# Staging DB to be used by AAT staging pod for functional tests
+module "bulk-scan-staging-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  name               = "${var.product}-${var.component}-staging"
+  product            = var.product
+  component          = var.component
+  location           = var.location_db
+  env                = var.env
+  database_name      = "bulk_scan_orchestrator"
+  postgresql_user    = "bulk_scanner_orch"
```
Comment (mdayican) [inline infrastructure/main.tf:58]: "bulk_scan_orchestrator" ?

Comment (apereverzin) [reply infrastructure/main.tf:58]: ok

**Review by mdayican — APPROVED** [review]

**Pull request merged**